project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 resolvers += "Flyway" at "https://flywaydb.org/repo"
 
-addSbtPlugin("org.flywaydb" % "flyway-sbt" % "3.2.1")
+addSbtPlugin("org.flywaydb" % "flyway-sbt" % "4.0.3")
```

Comment:
Turned out the issues I had were unrelated - removed the extra dependency in build.sbt